app/views/account/companies/show.html.slim

```diff
@@ -67,3 +67,18 @@
           .card-content
             .card-body
               = render 'account/companies/html/social_links', company: @company
+        - if @twitter.tweets.any?

```

Comment:
prefer `Object#present?` over `ActiveRecord::Relation#any?` to avoid extra queries